pyvera/__init__.py

```diff
@@ -645,3 +653,91 @@ def is_locked(self, refresh=False):
             self.refresh_complex_value('Status')
         val = self.get_complex_value('Status')
         return val == '1'
+
+class VeraThermostat(VeraDevice):
+    """Class to represent a thermostat."""
+    def get_payload(self, service_name, name, value):
+        return {
+            'id': 'lu_action',
+            'output_format': 'json',
+            'DeviceNum': self.device_id,
+            'serviceId': service_name,
+            'action': 'Set' + name,
+            'New' + name: value}
+
```

Comment:
This method looks quite generic - how are you using it? 

Don't think I can see it being used elsewhere in the class.

It looks very generic so my thought would be either to decide it's generally useful and add it into the base class - or drop. What do you think?